SwiftSyntaxDevUtils/Sources/swift-syntax-dev-utils/commands/LocalPrPrecheck.swift

```diff
@@ -0,0 +1,58 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import ArgumentParser
+import Foundation
+
+struct LocalPrPrecheck: ParsableCommand {
+  static let configuration = CommandConfiguration(
+    abstract: """
+      Ensure changes are fully tested, formatted, and validated before pull request submission.
+      """
+  )
+
+  @OptionGroup
+  var arguments: SourceCodeGeneratorArguments
```

Comment:
@ahoppen I've used this arguments group because it has all the needed arguments for this command but the name doesn't fit anymore, should I change it or stop using it and create local arguments?

Reply:
If you have any good new name ideas for `SourceCodeGeneratorArguments`. If you have any, I think renaming would be nice but otherwise, I think it’s OK as-is as well.